apps/platform-app/src/app/[locale]/(profile)/_actions/mutations/publishProfile.ts

```diff
@@ -6,6 +6,7 @@ import {
   updateProfileById,
 } from '@/backend/profile/profile.service'
 import { sendDiscordNotificationToModeratorChannel } from '@/lib/discord'
+import { runEvaluateProfileAgent } from '@/lib/langchain'
 import { withSentry } from '@/utils/errHandling'
 import { PublishingState } from '@prisma/client'
```

Comment:
**issue**: The `runEvaluateProfileAgent` function is called with an empty string as the first argument. Ensure that the correct profile data is passed for evaluation.